templates/terraform/schema_property.erb

```diff
@@ -15,7 +15,7 @@
 <% if tf_types.include?(property.class) -%>
 "<%= Google::StringUtils.underscore(property.name) -%>": {
   Type: <%= tf_types[property.class] %>,
-<%	if property.default.from_api -%>
+<%	if property.default&.from_api -%>
```

Comment:
I didn't really understand how `default` could be nil. But now I do.

Can you add `TODO(https://github.com/GoogleCloudPlatform/magic-modules/issues/167) Remove safe-access`

`@default ||= Provider::Terraform::Default.new` in the validate method here should ensure it is never nil.

The problem is that we ensure this method is always call even for nonoverriden properties. However, the logic is flawed and only ensures it is being called for top-level properties.

See issues #167 for more details.